server/common/metrics/constants.ts

```diff
@@ -17,6 +17,7 @@ export const COMPONENTS = [
   'notebooks',
   'trace_analytics',
   'metrics_analytics',
+  'integrations',
```

Comment:
I think this variable is not used anymore, can ignore this.

Reply:
It's used to collect usage metrics later on for API requests - I think we can keep this. Adding usage metrics in later PR